nixos/modules/services/desktops/espanso.nix

```
@@ -8,6 +8,13 @@ in {
   options = {
     services.espanso = {
       enable = mkEnableOption "Espanso";
+      wayland = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Use the Wayland compatible espanso package
+        '';
```

You can just make this a one liner
```suggestion
        description = "Whether to use the Wayland compatible Espanso package";
```

---

The module already uses this option and is currently broken because of it. I could have also removed the `mkIf` and we could allow people to set the wayland package manually, but there is a growing convention across other packages to support the wayland option.